lib/transfer_util.cu

```diff
@@ -704,23 +702,24 @@ namespace quda {
 
   template<typename Float>
   void BlockOrthogonalize(ColorSpinorField &V, int Nvec, 
```

Comment:
Is an extra template parameter needed here (spinBlockSize)?  Could one not just create a trait that maps from the nSpin template, to avoid having to pass this extra template around?